robot/src/main/java/us/ilite/robot/modules/Intake.java

```diff
@@ -25,8 +25,8 @@
     
     public Intake() {
         // Construction
-        spx1 = new VictorSPX(SystemSettings.kIntakeSPX1Address);
-        spx2 = new VictorSPX(SystemSettings.kIntakeSPX2Address);
+        spx1 = new VictorSPX(SystemSettings.kHatchIntakeSPXAddress);
```

Comment:
Also, why were these names changed? They weren't updated in the integration spreadsheet, and no one coordinated the changes with the integration team.